make/modules/java.desktop/Lib.gmk

```diff
@@ -35,10 +35,12 @@ $(call FillFindCache, $(wildcard $(TOPDIR)/src/java.desktop/*/native))
 ################################################################################
 # Create the AWT/2D libraries
 
-include lib/AwtLibraries.gmk
-include lib/ClientLibraries.gmk
+ifeq ($(call isTargetOs, android ios), false)
```

Comment:
This is acceptable for now, but I think we should try to find a more elegant way to resolve this.

Reply:
True. It's a complex issue and discussion though, but I agree we need to find an elegant way as a follow-up.